### ISSUE TITLE
Accessibility: html element doesn't have lang set on blog page

### ISSUE THREAD

**Issue by dgodongm** [mastodon/joinmastodon#365]
On https://blog.joinmastodon.org/, the html element doesn't have lang set. This is important for those using screenreaders.
See screenshot from Axe DevTools:
![Screenshot 2023-06-27 at 3 28 00 PM](https://github.com/mastodon/joinmastodon/assets/123777263/6dd98904-07f3-45c2-9e04-0ba97c616adc)

**Comment by varun-s22** [comment]
is this open? can i work on this?

**Comment by dgodongm** [comment]
@varun-s22 Issue is open and still happens. Don't see why it couldn't be worked on.

**Comment by varun-s22** [comment]
> @varun-s22 Issue is open and still happens. Don't see why it couldn't be worked on.

actually i dont see the blog file in the repository
i thought maybe its just another route, but localhost/blogs gave me 404

can you guide me, where to locate the file?

**Comment by dgodongm** [comment]
@varun-s22 Looks like it's actually a separate repo: https://github.com/mastodon/blog

**Comment by varun-s22** [comment]
> @varun-s22 Looks like it's actually a separate repo: https://github.com/mastodon/blog

seems like we need to migrate this issue then :)

**Comment by dgodongm** [comment]
@varun-s22 I don't appear to have permissions to move it so will just duplicate and close this

**Comment by dgodongm** [comment]
@varun-s22 Doesn't look like Issues are enabled for that repo so leaving here
